bindings/nodejs/lib/bindings.ts

```diff
@@ -27,6 +27,7 @@ const {
     getClientFromWallet,
     getSecretManagerFromWallet,
     migrateStrongholdSnapshotV2ToV3,
+    migrateDbFromChrysalisToStardust,
```

Comment:
Ignore if you want but we haven't put the From in the stronghold migration

Reply:
Removed it